CHANGELOG.md

```diff
@@ -1,12 +1,111 @@
-## v0.5.0 (Unreleased)
+## v0.5.0 (September xx, 2019)
 
  ### Notes
 
+ - This release is not backward compatible with the previous versions of Swarm. Please update your nodes.
+ - Ubuntu PPA support has been removed. We will add support for multiple software distribution platforms at a later stage. In the meanwhile you can still find the Swarm binaries at out [official download page](https://swarm-gateways.net/bzz:/swarm.eth/downloads/).
+
  ### Features
 
+ - **Content Pinning:** You can now pin content to your local swarm node. This way you can guarantee that certain content stays on your node. You can enable pinning by setting the `--enable-pinning` flag. For more information, read the [documentation](https://swarm-guide.readthedocs.io/en/latest/dapp_developer/index.html#pinning-content).
+
+The following features are **experimental** and are **not enabled by default on our public testnet**. If you want to test these features, make sure to use a separate network (e.g. create your own swarm cluster) and set the same feature flags on all nodes.
+
+ - **Push sync:** Additionally to our previous syncing algorithm, pull sync, we now also have "push sync" as an option available to synchronise the content across nodes. If you want to try it out, you have to start you swarm node with `--sync-mode=all`.
```

Comment:
In theory this sounds good, but we have found problems with `pushsync` and `pullsync` working together. Also if half of the nodes in a network enable `pushsync`, and the other half doesn't, we have an issue, as `pushsync` is not a standard devp2p protocol. Overall I am not sure if we should explain to people in the release notes how to try it out, as actually trying it out is a bit more involved.